
<launch>

<!-- Nodes to launch 360 lidar -->
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
  <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
  <!--param name="serial_baudrate"     type="int"    value="256000"--><!--A3 -->
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  </node>

  <node name="tf_360_lidar"  pkg="tf" type="static_transform_publisher" args="0 0 0 0 3.14 3.14 /base_link /laser 10" />


<!-- get puck detection node -->
<node pkg="wachl_puck_lidar" type="wachl_puck_lidar_node" name="get_puck_lidar" output="screen">
</node>


<!-- Load dynamic parameter -->
<node pkg="dynamic_reconfigure" name="dynamic_load_lidar" type="dynparam" args="load /puck_pick_laser $(find wachl_puck_lidar)/config/pickup_params.yaml" />


</launch>
